crypto/ocsp/ocsp_ext.c

```diff
@@ -449,12 +450,11 @@ X509_EXTENSION *OCSP_url_svcloc_new(X509_NAME *issuer, const char **urls)
             goto err;
         if ((ad->method = OBJ_nid2obj(NID_ad_OCSP)) == NULL)
             goto err;
-        if ((ad->location = GENERAL_NAME_new()) == NULL)
-            goto err;
         if ((ia5 = ASN1_IA5STRING_new()) == NULL)
             goto err;
         if (!ASN1_STRING_set((ASN1_STRING *)ia5, *urls, -1))
             goto err;
+        /* ad->location is init trougth ASN1_SIMPLE mechanism */
```

Comment:
Typo: through

Although I'm not entirely sure what you mean by "through ASN1_SIMPLE" mechanism. Presumably you mean it is created as part of `ACCESS_DESCRIPTION_new()`

Reply:
AFAI see : 
- any asn1 construct 
- who has field(s) declared using ASN1_SIMPLE
- field(s) that are non-POD type.
then,  asn1 mechanism is allocating them .
The leak detection tool says it happens in `tsan1_new.c`

I ask myself if this is a (new) feature or a very old bug ?